### DISCUSSION TITLE
biome's lsp server not seems to integrate formatter capability

### DISCUSSION THREAD

**Discussion by rtgiskard** [biomejs/biome#2787] [Q&A]
I'm neovim user running biome as LSP server, it does not seem to provide format capability, even though there is a sub command to format things. Or am I missing anything?

**Comment by rtgiskard** [comment]
https://github.com/biomejs/biome/discussions/87

**Comment by ematipico** [comment]
It should. We dropped dynamic capabilities. Are you using the latest version of Biome?

**Reply by rtgiskard** [reply]
Sorry for late response, forget to check for busy days ;(

```
$ nvim --version
NVIM v0.9.5
Build type: Release
LuaJIT 2.1.1713773202

   system vimrc file: "$VIM/sysinit.vim"
  fall-back for $VIM: "/usr/share/nvim"

Run :checkhealth for more info
```

the `biome_lsp` in use is `v1.7.3`

Current nvim in use does get `dynamicRegistration = false`, for which, from the link above (https://github.com/biomejs/biome/discussions/87), I have thought it's the reason format capability not enabled, so I need to use `conform` to declare biome as the formatter for json file.

It would be convenient if biome lsp server can register the related format capability, so that `lsp.format` will be out of the box, just like clangd.

Here is the LSP debug log on edit a JSON file, with no external formatter:
[lsp.log](https://github.com/biomejs/biome/files/15294080/lsp.log)
